packages/components/src/DateTimePickers/DateTime/Input/Input.component.js

```diff
@@ -1,22 +1,35 @@
 import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
 import DebounceInput from 'react-debounce-input';
 
 import { DateTimeContext } from '../Context';
+import { extractPartsFromDateTimeFormat } from '../date-extraction';
 
 export default function Input(props) {
 	const { datetime, inputManagement, errorManagement } = useContext(DateTimeContext);
+
 	return (
 		<DebounceInput
 			aria-describedby={errorManagement.inputErrorId}
 			autoComplete="off"
 			className="form-control"
 			debounceTimeout={300}
 			onFocus={errorManagement.onInputFocus}
-			type="text"
 			value={datetime.textInput}
 			{...inputManagement}
 			{...props}
+			type="text"
+			placeholder={extractPartsFromDateTimeFormat(inputManagement.placeholder, props.type)}
 		/>
 	);
 }
+
+Input.defaultProps = {
+	type: 'date',
```

Comment:
This is not what we as that would make some browser render their native picker. Moreover it is never pass to the input, as you changed the props order, overriding any type attribute by "text"